toonz/sources/translations/german/tnztools.ts

```diff
@@ -0,0 +1,1005 @@
+        <translation>Selektiv</translation>
+    </message>
+</context>
+<context>
+    <name>PinchTool</name>
+    <message>
+        <source>Size:</source>
+        <translation>Größe:</translation>
+    </message>
+    <message>
+        <source>Corner:</source>
+        <translation>Ecke:</translation>
+    </message>
+    <message>
+        <source>Manual</source>
+        <translation>Manuell</translation>
+    </message>
+</context>
+<context>
+    <name>PlasticTool</name>
+    <message>
+        <source>Build Skeleton</source>
+        <translation>Skelett aufbauen</translation>
+    </message>
+    <message>
+        <source>Paint Rigid</source>
+        <translation>Starr malen</translation>
+    </message>
+    <message>
+        <source>Animate</source>
+        <translation>Animieren</translation>
+    </message>
+    <message>
+        <source>Rigid</source>
+        <translation>Starr</translation>
+    </message>
+    <message>
+        <source>Flex</source>
+        <translation>Flexibel</translation>
+    </message>
+    <message>
+        <source>Mode:</source>
+        <translation>Modus:</translation>
+    </message>
+    <message>
+        <source>Vertex Name:</source>
+        <translation>Vertex-Name:</translation>
+    </message>
+    <message>
+        <source>Allow Stretching</source>
+        <translation>Strecken erlauben</translation>
+    </message>
+    <message>
+        <source>Thickness</source>
+        <translation>Dicke</translation>
+    </message>
+    <message>
+        <source>Keep Distance</source>
+        <translation>Abstand beibehalten</translation>
+    </message>
+    <message>
+        <source>Global Key</source>
+        <translation>Globale Taste</translation>
+    </message>
+    <message>
+        <source>A group of skeletons already exists for current column. Replacing it will also substitute any existing vertex animation.
+
+Do you want to continue?</source>
+        <translation>Eine Gruppe Skelette existiert bereits für die aktuelle Spalte. Sie zu ersetzen, wird jegliche vorhandenen Vertex-Animationen austauschen.
+
+Möchten Sie fortfahren?</translation>
+    </message>
+    <message>
+        <source>Ok</source>
+        <translation>OK</translation>
+    </message>
+    <message>
+        <source>Cancel</source>
+        <translation>Abbrechen</translation>
+    </message>
+    <message>
+        <source>Copy Skeleton</source>
+        <translation>Skelett kopieren</translation>
+    </message>
+    <message>
+        <source>Paste Skeleton</source>
+        <translation>Skelett einfügen</translation>
+    </message>
+    <message>
+        <source>Delete Vertex</source>
+        <translation>Vertex löschen</translation>
+    </message>
+    <message>
+        <source>Set Key</source>
+        <translation>Taste setzen</translation>
+    </message>
+    <message>
+        <source>Set Rest Key</source>
+        <translation>Rast-Taste setzen</translation>
```

Comment:
Using "Ruhepositions-Key setzen" until something better comes along and remains consistent